lib/util/ansi.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
Should we add this to `sister-ansi` instead?

Reply:
I think I've already created a PR for that on sisteransi.

Reply:
Awesome! I somehow missed that. It's merged now!